backbone/src/main/javaorg/researchstack/backbone/model/User.java

```diff
@@ -0,0 +1,85 @@
+package org.researchstack.backbone.model;
+
+import java.io.Serializable;
+import java.util.Date;
+
+/*
+  Created by bradleymcdermott on 10/22/15.
+ */
+public class User implements Serializable
```

Comment:
I'm not sure why the git history didn't transfer for this file, but not sure it is that important.